node-cli/index.js

```diff
@@ -21,8 +30,10 @@ async function openRealm() {
     },
     path: "localRealmDb/tracker",
   };
-
-  realm = new Realm(config);
+  realm = Realm.open(config);
+} catch(e) {
+    output.error(e);
```

Comment:
Nitpick: could be useful to add a message to prefix the error with to describe what type of error might be reported here.